debian/packages.d/systemd.customise.add/init.d/20confd.sh

```diff
@@ -0,0 +1,39 @@
+# Apply configurations from conf.d
+#
+# Copyright (C) 2018 Benedict Lau <benedict.lau@groundupworks.com>
+
+CONFDIR=conf.d
+
+try_partition() {
+    # Create file system node from partition
+    #mknod /dev/$1 b $2 $3 2>/dev/null
+
+    # Mount partition as read-only
+    mount /dev/$1 /mnt -o ro
+
+    # Find conf.d in root of partition and extract in alphabetical order each
+    # tar.gz archive into the ramdisk /etc, overwriting any existing file
+    if [ -d "/mnt/$CONFDIR" ]; then
+        confs=$(find /mnt/$CONFDIR -name *.tar.gz)
+        for conf in $confs; do
+            echo Applying configurations from /dev/$1: $conf
```

Comment:
Why not just use "for conf in /mnt/$CONFDIR/*.tar.gz" ?

Using find will traverse subdirs and is not known to be sorted